lms/djangoapps/certificates/models.py

```diff
@@ -945,6 +953,24 @@ def set_language_specific_templates_enabled_for_course(cls, course_key, is_enabl
             defaults=default
         )
 
+    @classmethod
+    def is_hours_of_effort_included_for_course(cls, course_key):
+        """Check whether hours of effort are meant to be included in a course's certificates
+
+        Arguments:
+            course_key (CourseKey): The identifier for the course.
+
+        Returns:
+            boolean
+
+        """
+        try:
+            latest = cls.objects.filter(course_key=course_key).latest()
+        except cls.DoesNotExist:
+            return False
```

Comment:
This conflicts with the default value set in the database migration. I would think that if we didn't have a certificate configuration row for a particular course, the behavior for hours of effort would be the same as if we did have a configuration row but didn't set include_hours_of_effort.

Reply:
Perhaps, then, the default actually ought to be False, because I don't think we would want to default to include Hours of Effort if there isn't a CertificateGenerationCourseSetting for the run.